src/features/game/types/expansions.test.ts

```
@@ -406,3 +409,213 @@ describe("getExpectedResources", () => {
+    const req = getExpansionRequirements({
+      island: "swamp",
+      expansion: 42,
+      ascensionLevel: 2,
+    });
+    expect(req?.resources).toEqual({ Crimstone: 210, Oil: 560, Obsidian: 42 });
+    expect(req?.coins).toBe(105000);
+    expect(req?.seconds).toBe(84 * HOUR);
+  });
+
+  it("treats ascensionLevel 0 the same as no ascensionLevel (static path)", () => {
+    const noAscension = getExpansionRequirements({ island: "basic", expansion: 4 });
```

ESLint

[error] Replace `·island:·"basic",·expansion:·4` with `⏎······island:·"basic",⏎······expansion:·4,⏎···`

(prettier/prettier)